### PR TITLE
Serve a robots.txt file

### LINKED ISSUE #293: Create a robots.txt file
We should create a [robots.txt file ](https://support.google.com/webmasters/answer/6062596?hl=en)

### PR DESCRIPTION
**Pull request by rviscomi**
Fixes #293 
Progress on #318 

I created an empty sitemap.xml file to show how it can be served, similar to robots.txt. We still need to populate the file. If it's dynamically generated we'll need a different approach.

### CODE REVIEW

File: src/static/robots.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+User-agent: *
```
Comment (AymenLoukil) [inline src/static/robots.txt]: Ensure its UTF-8 encoded

Comment (rviscomi) [reply src/static/robots.txt]: Is the action item to explicitly serve a content encoding header with this file? Or is there something else we need to do?

Comment (AymenLoukil) [reply src/static/robots.txt]: Just to be sure when creating the txt file, should be UTF-8 encoded. In the answer header we should have : content-type: text/plain; charset=utf-8

Comment (rviscomi) [reply src/static/robots.txt]: Ok yeah Flask returns a header with this charset.

> `Content-Type: text/plain; charset=utf-8`

**Comment by rviscomi** [comment]
I've removed the sitemap parts of this PR since @bazzadp is working on that in #345

**Pull request merged**